prediction_market_agent/agents/safe_guard_agent/safe_utils.py

```diff
@@ -108,13 +109,14 @@ def sign_or_execute(safe: Safe, tx: SafeTx, api_keys: APIKeys) -> TxParams | Non
     threshold = safe.retrieve_threshold()
 
     if threshold > len(tx.signers):
-        logger.info("Threshold not met yet, just adding a sign.")
+        logger.info("Threshold not met yet, posting a signature.", stremalit=True)
         api = TransactionServiceApi(EthereumNetwork(RPCConfig().chain_id))
```

Comment:
I think we should also analyze Ethereum transactions, since they probably are quite similar.

I see that in `get_safe_history` (https://github.com/gnosis/prediction-market-agent/blob/a8fe16669550a852ceff92f5a5e8a3903dc2d8e8/prediction_market_agent/agents/safe_guard_agent/safe_api_utils.py#L142) the `chain_id` parameter is being fetched from .env.
Would it be too much work to add a dropdown to Streamlit allowing Ethereum (chain_id 1) to also be fetched?

Reply:
No idea what will happen if we change chain id in the environment, we never did that so far 😄 But that's a problem for another day: https://github.com/gnosis/prediction-market-agent/issues/848